rootfs/api/tests/test_hooks.py

```diff
@@ -105,6 +106,28 @@ def test_key_hook(self):
             {'key': public2, 'fingerprint': '43:fd:22:bc:dc:ca:6a:28:ba:71:4c:18:41:1d:d1:e2'}
         ]})
 
+        # Fetch a valid ssh key
+        url = '/v2/hooks/key/{}'.format(base64.urlsafe_b64encode(public.encode('utf-8')).decode())
```

Comment:
I believe that using the encoded key is good enough for now, but we should, in the future, switch to a fingerprint. Doing so will at least further obfuscate identifying information in the public key. https://github.com/deis/builder/issues/158 tracks